utils/_context/_scenarios/__init__.py

```diff
@@ -387,6 +387,25 @@ def all_endtoend_scenarios(test_object):
         scenario_groups=[ScenarioGroup.APPSEC],
     )
 
+    appsec_meta_struct_enabled = EndToEndScenario(
```

Comment:
If I'm not wrong, that's the default behavior? If yes, we can run those test under the default scenario ?

Reply:
Yep sorry! It was a leftover from an older commit thanks 😄.
I deleted it in e93f678
All tests with meta struct enabled are run in the default scenario.